src/it/java/org/checkstyle/checks/imports/ImportOrderTest.java

```diff
@@ -0,0 +1,96 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2018 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package org.checkstyle.checks.imports;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.Test;
+
+import com.google.checkstyle.test.base.AbstractModuleTestSupport;
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+import com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck;
+import com.puppycrawl.tools.checkstyle.utils.CommonUtil;
+
+public class ImportOrderTest extends AbstractModuleTestSupport {
+
+    protected final String getNonCompilablePath(String filename) throws IOException {
```

Comment:
This method should be in `AbstractModuleTestSupport`.

Reply:
done. Moved to `AbstractPathTestSupport`